docs/reference/glossary.rst

```diff
@@ -0,0 +1,97 @@
+.. _glossary:
+
+========
+Glossary
+========
+
+.. contents::
+  :local:
+
+Flocker release types
```

Comment:
We don't do "Beta" releases, so "Beta" isn't really a release type. See `docs/gettinginvolved/infrastructure/release-schedule-versioning.rst` for our release types.